submissions/tosinIshola/html-forms/contact_form.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>CONTACT FORM</title>
+</head>
+<body>
+
+<nav>
+    <ul>
+        <li><a href="">HOME</a> </li>
+        <li><a href="">ABOUT US</a></li>
+        <li><a href="">OUR PRODUCT</a> </li>
+        <li><a href="" > CUSTOMER SERVICE</a> </li>
+    </ul>
+
+</nav>
+
+<h1>Contact Form</h1>
+
+<form action="#" method="post" enctype="multipart/form-data">
```

Comment:
All this `action="#" method="post" enctype="multipart/form-data"` aren't needed. Can you remove them?